FastSimulation/Tracking/plugins/SimTrackIdProducer.h

```diff
@@ -26,13 +26,16 @@ class SimTrackIdProducer : public edm::stream::EDProducer <>
 
       virtual void produce(edm::Event& e, const edm::EventSetup& es) override;
 
-      std::vector<int> SimTrackIds;
 
 private:
 
       // consumes 
       edm::EDGetTokenT<reco::TrackCollection>  trackToken;
-
+      double maxChi2_;
+      std::vector< edm::EDGetTokenT<edm::ValueMap<int> > > overrideTrkQuals_;
+      bool filterTracks_ = false;
+      reco::TrackBase::TrackQuality trackQuality_;
+     
 };
 
 #endif
```

Comment:
Keeping the quality & chi2 criteria actually makes sense.
In FullSim it depends on chi2 and quality whether the hits of a track are masked or not, see e.g.
http://cmslxr.fnal.gov/lxr/source/RecoTracker/IterativeTracking/python/PixelPairStep_cff.py#0005
If, in a later pr, we change the name of the module from SimTrackIdProducer to e.g. TrackMaskProducer there will be no confusion.